### PR TITLE
Creating .dockerignore

### PR DESCRIPTION
**Pull request by burgwyn**
We'll merge this into `dev` then add it to our stg -> master PR

### CODE REVIEW

**Comment by afrasier** [comment]
Looks fine to me, but I'm not a docker expert.

**Comment by burgwyn** [comment]
This is going to be a fun standoff since I'm no Python expert.  Are there any other artifacts in the repo that we don't want to publish?

**Comment by afrasier** [comment]
I'm not sure - would the docker image need the virtual environment stuff to actually run?

**Comment by afrasier** [comment]
Then again all checks have passed so I suppose it's happy. I'd run it by James to ensure we've fixed the issue he had with it.

File: Dockerfile [hunk 1]
```diff
@@ -8,6 +8,9 @@ RUN apt-get update && apt-get install -y postgresql-client
 RUN mkdir /app
 WORKDIR /app
 
-ADD . /app/
+COPY talentmap_api /app/talentmap_api/
+COPY logs /app/logs/
```
Comment (jseppi) [inline Dockerfile]: Is the `logs` directory necessary?

Comment (afrasier) [reply Dockerfile]: Needs to exist, but doesn't need anything in it.

Comment (jseppi) [reply Dockerfile]: Cool -- maybe we should add it to `.dockerignore` and instead add another line with `RUN mkdir /app/logs` That way no potentially weird/sensitive logs make it into the image.

**Pull request merged**